packages/mask/src/extension/popups/pages/Wallet/ConnectWallet/index.tsx

```diff
@@ -108,7 +113,7 @@ const ConnectWalletPage = memo(() => {
                 },
             })
         },
-        [isLocked, getLockStatusLoading],
+        [isLocked, getLockStatusLoading, wallets],
```

Comment:
Only care about whether there are wallets or not.

```suggestion
        [isLocked, getLockStatusLoading, !wallets.length],
```